t/plugin/udp-logger.t

```diff
@@ -355,45 +362,11 @@ GET /t
 passed
 
 
-
-=== TEST 10: access
---- stream_conf_enable
---- extra_stream_config
-    server {
-        listen 8125 udp;
-        content_by_lua_block {
-            local decode = require("toolkit.json").decode
-            ngx.log(ngx.WARN, "the mock backend is hit")
-
-            local sock, err = ngx.req.socket()
-            if not sock then
-                ngx.log(ngx.ERR, "failed to get the request socket: ", err)
-                return
-            end
-
-            local data, err = sock:receive()
-
-            if not data then
-                if err and err ~= "no more data" then
-                    ngx.log(ngx.ERR, "socket error, returning: ", err)
-                end
-                return
-            end
-
-            data = decode(data)
-            assert(data.client_ip == "127.0.0.1")
-        }
-    }
---- request
-GET /hello
---- response_body
-hello world
---- wait: 2
---- error_log
-the mock backend is hit
---- no_error_log
-[error]
-
+=== TEST 10: log format in plugin_metadata
```

Comment:
3 blank line in need to make linter happy. :)

Reply:
done